util/object.go

```diff
@@ -0,0 +1,91 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MakeClusterRole makes a ClusterRole object using the provided
+// name for the object's name.
+func MakeClusterRole(name string) *rbacv1.ClusterRole {
```

Comment:
My first thought was that `Make` was actually creating the object in Kubernetes. I think it could work but would `Generate` be more clear? This is a super nit thing, so no biggie. Names are hard. 😢  Just thinking out loud.

Reply:
I'll change the prefix Make -> Generate.